src/utils.c

```diff
@@ -42,4 +45,76 @@ char *find_available(char *filename) {
     return path;
 }
 
+/* \brief get a substring from corrected call to repeat it
+ *
+ * \returns a substring based on the sent and corrected callsign
+ */
+void get_partial_callsign(char *call1, char *call2, char *partial) {
```

Comment:
This function needs to be covered by tests.

Reply:
Commit 461398e replaced by your solution. Thanks. Please resolve this conversation if you agree the modification

Reply:
Will you add the tests from the other repo or shall I do it?

Reply:
Uhm, sorry, which tests are you thinking about?

Reply:
added in 079d6cd

Reply:
Thanks!